src/styles.css

```diff
@@ -99,3 +99,41 @@ markdown pre code {
 .starring-button {
   margin-top: 20px;
 }
+
```

Comment:
These following styles are presumably referenced by the external content, right? If so, I think you should do one of the following:

1. You should have comments before and after this section stating that it used by the external content, or someday somebody like me will come in and delete it  thinking it's not being used because I can't find usages in grep.
2. More generally, I wonder if this styling should instead live in the external content itself, perhaps as inline css. It feels slightly off to have it separated from HTML. What if we decide one day that we don't want the external content to be cards? You'd either have to change this code, requiring a new UI build, which is what we're trying to avoid, or you change the external HTML to not use these classes, in which case, you "orphan" this CSS. OTOH, there are libraries of CSS that live independently from the HTML. And there's the risk that the CSS in the external HTML might clobber our CSS.

@garyluu thoughts on this?

Reply:
For #2 I was thinking about that, but it gots a bit unruly to have everything inline. I was thinking cards could be our basic default, but if we did want to do a one-off styling for that specific html we could do in-line styles.

Reply:
After further verbal discussion, I don't think option 2 makes sense. We're really loading a snippet of HTML, not an HTML page, where I had been thinking you could you have the CSS in the `<head>` section. So option 1 is the way to go.

Reply:
A few things:
- I didn't realize we still had a style.css file
- If possible, move this into the scss file and then have one class style that contains a bunch of nested styles using scss. This way u can just comment on the entire thing.
- I thought we agreed to ban 'px' for most things
- I don't mind having the style in this repo as long as it's obvious (comments and such)